acceptance/bundle/variables/var_file_overrides/output.txt

```diff
@@ -0,0 +1,126 @@
+
+>>> $CLI bundle validate -o json --var-file=var_files/normal.json
+{
+  "job_cluster_key": "mlops_stacks-cluster",
+  "new_cluster": {
+    "node_type_id": "Standard_DS3_v2",
+    "num_workers": 2
+  }
+}
+
+>>> $CLI bundle validate -o json --var=cluster_key=mlops_stacks-cluster
+{
+  "job_cluster_key": "mlops_stacks-cluster",
+  "new_cluster": {
+    "node_type_id": "default",
+    "num_workers": 1
+  }
+}
+
+>>> errcode $CLI bundle validate -o json --var-file=var_files/normal.json --var=cluster_key=mlops_stacks-cluster
+Error: cannot specify both --var and --var-file flags
+
+
+Exit code: 1
+{
+  "job_cluster_key": "${var.cluster_key}",
+  "new_cluster": {
+    "node_type_id": "${var.cluster.node_type_id}",
+    "num_workers": "${var.cluster_workers}"
+  }
+}
+
+>>> errcode $CLI bundle validate -o json --var-file=var_files/not_found.json
+Error: failed to read variables file: open var_files/not_found.json:<stripped>
+
+
+Exit code: 1
+{
+  "job_cluster_key": "${var.cluster_key}",
+  "new_cluster": {
+    "node_type_id": "${var.cluster.node_type_id}",
+    "num_workers": "${var.cluster_workers}"
+  }
+}
+
+>>> errcode $CLI bundle validate -o json --var-file=var_files/invalid_json.json
+Error: failed to parse variables file: error decoding JSON at :0:0: invalid character 'o' in literal false (expecting 'a')
+
+
+Exit code: 1
+{
+  "job_cluster_key": "${var.cluster_key}",
+  "new_cluster": {
+    "node_type_id": "${var.cluster.node_type_id}",
+    "num_workers": "${var.cluster_workers}"
+  }
+}
+
+>>> errcode $CLI bundle validate -o json --var-file=var_files/wrong_file_structure.json
+Error: failed to parse variables file: var_files/wrong_file_structure.json:1:1: expected a map, found a sequence
+  in var_files/wrong_file_structure.json:1:1
```

Comment:
This location is always the same because it is taken from the root value. Can be omitted.